lib/portage/util/env_update.py

```diff
@@ -294,6 +295,16 @@ def _env_update(makelinks, target_root, prev_mtimes, contents, env, writemsg_lev
         except IOError as e:
             if e.errno != errno.ENOENT:
                 raise
+    else:
```

Comment:
Lines from 240 (the comment before `if prelink_capable:`) to 296 can be replaced with single line:
```
    if not prelink_capable:
```

Reply:
It has been done. Thanks for clarifying earlier! 😎